css/owned-card.styl

```diff
@@ -1,3 +1,18 @@
+.discipline-section
```

Comment:
Now that owned card is collections/favourites etc only and not used in project cards, we should not be changing this file...